about-me.html

```diff
@@ -0,0 +1,142 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8"> 
+	<title>Eleanor Stevens</title>
+	<link href="portfolio-stylesheet.css" type="text/css" rel="stylesheet">
+	<link href="normalize.css" type="text/css" rel="stylesheet">  <!-- this means the style is universal across all browsers -->
+	<link href="DINNextLTPro-Regular.ttf" type="text/ttf" rel="stylesheet">
+	<link href="https://github.com/elstevens23" rel="stylesheet">
+	
+</head>
+<body>
+<section class="navBar">
+	<nav>
+		<a href="portfolio.html">HOME</a>
+		<a href="about-me.html">ABOUT ME</a>
+		<a href="web-portfolio.html">PORTFOLIO</a>
+		<a href="contact.html">CONTACT ></a>
+	</nav>
+	
+	<h1 class="pageTitle"> <a href="portfolio.html">Eleanor Stevens</a> </h1>
+	<!-- <h2 class="subheading">WEBSITE DESIGN & DEVELOPMENT</h2> -->
+	<div class="right">
+	</div>
+	
+</section>
+
+<section class="aboutmepage">
+	<div class="top-aboutme">
+		<img src="img_avatar2.jpg" alt="Woman cartoon avatar">
+		<div class="aboutme-heading">
+			<h1><strong>About Me</strong></h1>
+		<p>Hi, my name is Eleanor and I am training to be
+		a full-stack developer. I am currently studying 
+		at Mayden Academy, in Bath.</p>
+		</div> 
+	</div>
+	
+	<div class="middle-aboutme">
+		<div class="aboutheading-two">
+			<h1>What I do</h1>
+		</div>
+		
+		<div class="about-grid">
+			
+			<div class="column">
+				<img src="https://img.icons8.com/ios/50/000000/for-experienced.png" alt="Head Gear Cog Icon"/>
```

Comment:
its not a good idea to link to images on the web, you should download them and link to them from your computer

Reply:
This was the site that I took the icons from but needed to mention them in my footer so that I wasn't copyrighted. Can I still change it to a saved file or should I keep the link like they told me to, in order to avoid copyright?